test/testdrive/subexpression-replacement.td

```diff
@@ -53,7 +53,7 @@ $ set-regex match=(\s\(u\d+\)|\n|materialize\.public\.) replacement=
 # Nested OR/disjunctions are not optimized
 
 > EXPLAIN SELECT * FROM t1 WHERE col_null IS NULL OR (col_null IS NULL OR col_not_null = 5);
-"%0 =| Get t1| Filter ((#0) IS NULL || ((#0) IS NULL || (#1 = 5)))"
+"%0 =| Get t1| Map (#0) IS NULL| Filter (#2 || (#2 || (#1 = 5)))| Project (#0, #1)"
```

Comment:
When you see Map-Filter-Project sequences change, there should actually be no performance difference because consecutive Maps, Filters, and Projects in MIR all get collapsed into a `MapFilterProject` object at the LIR stage and reoptimized.